utils/zipdir.py

```diff
@@ -9,7 +9,7 @@ def zipdir(directory, compression=zipfile.ZIP_DEFLATED):
     zipname = '%s.zip' % directory
     prefix = os.path.dirname(directory)
     with zipfile.ZipFile(zipname, 'w', compression) as zf:
-        for dirpath, dirnames, filenames in os.walk(directory):
+        for dirpath, _, filenames in os.walk(directory):
```

Comment:
I'd rather not use `_` for unused variables since we also use `gettext` to install `_()` as a builtin for translating strings. (There's only one use left of it for this in the Stentura code, other instances have been removed).